### PR TITLE
Sparta summary

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/special/players/spart.xml [hunk 1]
```diff
@@ -4,7 +4,12 @@
   <Identity>
     <Civ>spart</Civ>
     <GenericName>Spartans</GenericName>
-    <History>Sparta was a prominent city-state in ancient Greece, and its dominant military power on land from circa 650 BC. Spartan culture was obsessed with military training and excellence, with rigorous training for boys beginning at age seven. Thanks to its military might, Sparta led a coalition of Greek forces during the Greco-Persian Wars, and won over Athens in the Peloponnesian Wars, though at great cost.</History>
+    <History>
+        Sparta was a prominent city-state in Ancient Greece, and its dominant military power on from the seventh century BC. It peaked after the Peloponnesian Wars in the late fifth century BC but declined soon after, ending when they lost to Thebes in the battle of Leuctra in 371BC.
+        Spartan culture was obsessed with military training and excellence, all spartan boys were drafted into rigorous training at age seven, and their military service lasted until they were sixty. All trades were done by their serfs, the Helots.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/special/players/spart.xml:9]:
```suggestion
        Spartan culture was obsessed with military training and excellence, all Spartan boys were drafted into rigorous training at age seven, and their military service lasted until they were sixty. All trades were done by their serfs, the Helots.
```

File: 0-ad-history-mod-package/simulation/templates/special/players/spart.xml [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
+        Spartans lived a strict and minimalistic lifestyle, separate from many of the other civilization, due to reforms such as iron money, which were given by Lycurgus.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/special/players/spart.xml:10]:
```suggestion
        Spartans lived a strict and minimalistic lifestyle, separate from many of the other civilizations, due to reforms such as iron money, which were given by Lycurgus, the legendary Spartan lawgiver.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by TheShadowOfHassen** [comment]
Can I merge something?

**Comment by indoptogopt** [comment]
Yes you can merge it

**Pull request merged**